Tests/ItemProviderTests.swift

```diff
@@ -40,6 +40,7 @@ class ItemProviderTests: XCTestCase {
     
     // MARK: - Item Provider Item Handler Tests
 
+    @MainActor
```

Comment:
Can we mark `ItemProviderTests` as `@MainActor` instead of each of these individual functions?